### PR TITLE
Chaninverse os

### PR DESCRIPTION
**Pull request by jose-zamora**
# What changed
https://www.loom.com/share/35a3e50b1f75495f898c093da80008f1
# How did you test the change?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "chromatic": "npx chromatic --project-token=0edcf7bf162e"
   },
   "dependencies": {
+    "@chainverse/os": "^0.1.5",
```
Comment (clemp) [inline package.json:17]: What do we need to do to keep this version consistent with the release version in the `chainverse-os` Github repo?

https://github.com/DiamondDAO/chainverse-os

File: src/pages/_app.page.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import "@/styles/globals.css";
+import '@chainverse/os/dist/chainverse-os.css';
```
Comment (clemp) [inline src/pages/_app.page.tsx:2]: When I test this branch locally I get the following error related to this line 
image

File: src/pages/explorer/index.page.tsx [hunk 3]
```diff
@@ -6,30 +6,57 @@ import {
...
-import * as styles from "./styles";
+} from '@/services/Apollo/Queries';
+import { SearchIcon } from '@chakra-ui/icons';
+import { useSpring, a } from 'react-spring';
+import Fuse from 'fuse.js';
+import { filterUniqueObjects } from '@/common/utils';
+import Router from 'next/router';
+import { bodyText } from '@/theme';
+import * as styles from './styles';
+import { SearchOS, SearchTypes } from '@chainverse/os';
```
Comment (clemp) [inline src/pages/explorer/index.page.tsx:25]: I don't see `SearchTypes` as a comonent in the `chainverse-os` repo: https://github.com/DiamondDAO/chainverse-os/tree/main/src/components

Should I be looking somewhere else?

**Review by clemp — CHANGES_REQUESTED** [review]
I tried to run this branch locally and had an error (details in comments), and then I commented another question about the `SearchTypes` which was referenced in this PR but I couldn't find it in the `chainverse-os` repo.

File: src/pages/_app.page.tsx (hunk 2 above)
Comment (clemp) [reply src/pages/_app.page.tsx:2]: Disregard - I had to run `npm i` to reinstall dependencies

**Review by clemp — CHANGES_REQUESTED** [review]
When I use the Explore page and type in a search term (I've connected the API to the Chainverse application database) I get this error:

image

File: src/pages/explorer/index.page.tsx (hunk 3 above)
Comment (jose-zamora) [reply src/pages/explorer/index.page.tsx:25]: image
@clemp searchType its not a component. as the name its a Type

File: package.json (hunk 1 above)
Comment (jose-zamora) [reply package.json:17]: its not needed keep consistent with the release version, bc one thing the the version we have published in github and other the version we are still working on

**Comment by jose-zamora** [comment]
@clemp based on the last screenshot you are not enabled the Search component you are still using the old implementation. Remember you need to enable the `feature flag` by running some commands in the console
- window.showChainverseOS()
- window.removeChainverseOS()
here is the loom video about how you can enable it
https://www.loom.com/share/35a3e50b1f75495f898c093da80008f1

**Comment by clemp** [comment]
> * window.showChainverseOS()

Got it, thank you. I did run this command and saw the app rebuild with the new Search OS component.

However, when connecting to the application database (the one used for `staging.chainverse.diamonds`) I still do get this error: 
image

I think this may be related to this error: 
image

I'll investigate the data issue and update here.

**Pull request merged**